package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-preferences",
-  "version": "0.0.0",
+  "version": "10.0.0",
```

Comment:
Please do not change this one. Change the Library one to be published.

Reply:
Please change this one: https://github.com/systelab/systelab-preferences/blob/angular-13-upgrade/projects/systelab-preferences/package.json